DNA_RNA

```diff
@@ -0,0 +1,40 @@
+class DNA:
+    def __init__(self, sequence):
+        self.sequence = sequence
+        self.sequence_length = len(sequence)
+
+    def gc(self):
+        gc_counter = 0
+        for i in range(self.sequence_length):
+            if (self.sequence[i] == 'G') or (self.sequence[i] == 'C'):
+                 gc_counter += 1
+        gc_content = round(float((gc_counter / self.sequence_length) * 100), 1)
+        return gc_content
+
+    def reverse_complement(self):
+        dna_reverse_complement_dict = {'A': 'T', 'C': 'G', 'G': 'C', 'T': 'A'}
+        return ''.join([dna_reverse_complement_dict[base] for base in sequence[::-1]])
```

Comment:
Probably, for the function to work, `sequence` must be substituted to `self.sequence` here.

Reply:
Added self. to the "sequence"